src/main/java/lamball/Storage.java

```diff
@@ -0,0 +1,193 @@
+package lamball;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+
+/**
+ * Contains methods that pertain to file access and modification.
+ *
+ * @author ongzhili
+ */
+public class Storage {
+
+    protected String filePath = "src/main/java/data/list.txt";
+    protected String tempFilePath = "src/main/java/data/tempfile.txt";
+
+    /**
+     * Obtains and initializes list from saved text file locally.
+     *
+     * @param lamb Initialized Lamball chatbot instance.
+     * @return Text for the GUI to display.
+     */
+
+    public static String obtainSavedFile(Lamball lamb) {
+        String returnVal = "";
+        File folder = new File("src/main/java/data");
+        if (!folder.exists()) {
+            System.out.println("Folder does not exist. Creating folder...");
+            returnVal += "Folder does not exist. Creating folder...\n";
+            folder.mkdirs();
+        }
+        File savedList = new File("src/main/java/data/list.txt");
+        try {
+            // Try to create file
+            if (savedList.createNewFile()) {
+                System.out.println("List created successfully at: " + savedList.getAbsolutePath());
+                returnVal += "List created successfully at: " + savedList.getAbsolutePath() + "\n";
+            } else {
+                System.out.println("Seems like I haave helped you before, so no new list is needed!");
+                returnVal += "Seems like I haave helped you before, so no new list is needed!\n";
+            }
+            initializeListFromText(savedList, lamb);
+        } catch (IOException e) {
+            System.err.println("An error occurred while creating the file: " + e.getMessage());
+            returnVal += "An error occurred while creating the file: " + e.getMessage();
+        } finally {
+            return returnVal;
+        }
+    }
+
+    private static void initializeListFromText(File savedList, Lamball lamb) throws FileNotFoundException {
```

Comment:
Perhaps the naming of the methods can be more consistent here? We have:
1. Load = obtainSavedFile
2. Save = writeToFile (There's method overloading as well, might want to consider commenting or not overload if the methods are doing different things.)